### PR TITLE
[READY]IMMERSIVE ROLEPLAY - Mobs now turn towards and are shifted based on grab state towards something pulling them

### PR DESCRIPTION
**Pull request by silicons**
In other words, you can now dance. 
@subject217 
:cl:
experimental: Mobs now turn towards anything grabbing them when they move.
experimental: Mobs are shifted towards anything grabbing them based on move state too. You can now dance by grabbing each other!
/:cl:

### CODE REVIEW

**Comment by kevinz000** [comment]
lemme just slap wip on this as it's not tested

**Comment by optimumtact** [comment]
I don't understand what is taking you so long

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -851,13 +851,13 @@
 				if(!riding_datum.equip_buckle_inhands(M, 2))	//MAKE SURE THIS IS LAST!!
 					M.visible_message("<span class='warning'>[M] can't climb onto [src]!</span>")
 					return
-			. = ..(M, force, check_loc)
 			stop_pulling()
+			. = ..(M, force, check_loc)
```
[the remark this answers is not in the document]
Comment (optimumtact) [reply code/modules/mob/living/carbon/human/human.dm]: did you test it?

**Comment by optimumtact** [comment]
Did you really bundle an unrelated fix in with a feature?

**Comment by anconfuzedrock** [comment]
Muh sleeping carp back kicks!

**Pull request merged**